Makefile

```diff
@@ -119,6 +119,9 @@ test:
 test-conformance:
 	python conformance/run_conformance.py http://localhost:18201
 
+test-authorization:
+	python conformance/run_conformance.py http://localhost:18201 -u testuser -p abc123
```

Comment:
Auth is going to have a lot of different pass/failure states then regular conformance. Maybe it should be written as a separate script.